api/V1/util.js

```diff
@@ -67,14 +67,15 @@ function addRecordingFromPost(model, request, response) {
 
       modelInstance.set('DeviceId', device.id);
       modelInstance.set('GroupId', device.GroupId);
-      if (typeof device.public == 'boolean')
+      if (typeof device.public == 'boolean') {
```

Comment:
I could start a fight about the lack of brackets...I prefer brackets for one line statements because it means if someone adds a second line it doesn't break.   This could especially be a problem since we are working with python at the same time...

Reply:
I completely agree with you! We should always use curly brackets in JS even when not strictly required. In fact our eslint config even warns about this (we'll start running eslint automatically on pull requests soon once the code is in slightly better shape).

See:
* https://github.com/TheCacophonyProject/cacophony-api/blob/master/.eslintrc.yml#L16
* https://github.com/TheCacophonyProject/cacophony-api/issues/33

I didn't actually touch this line myself. I suspect my editor just fixed up trailing whitespace.

Reply:
Sorry I got it round the wrong way!